src/data/projects/newrelic-ruby-agent.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "newrelic-ruby-agent",
+  "fullName": "newrelic/newrelic-ruby-agent",
+  "slug": "newrelic-ruby-agent",
+  "owner": {
+    "login": "newrelic",
+    "type": "Organization"
+  },
+  "title": "Ruby Agent",
+  "supportUrl": "https://discuss.newrelic.com/c/support-products-agents/ruby-agent",
+  "githubUrl": "https://github.com/newrelic/newrelic-ruby-agent",
+  "permalink": "https://opensource.newrelic.com/projects/newrelic/newrelic-ruby-agent",
+  "iconUrl": null,
+  "shortDescription": "New Relic's Ruby Agent",
+  "description": "The New Relic Ruby agent monitors your applications to help you identify and solve performance issues. You can also extend the agent's performance monitoring to collect and analyze business data to help you improve the customer experience and make data-driven business decisions.",
+  "ossCategory": "community-project",
+  "primaryLanguage": "Ruby",
+  "projectTags": ["agent", "ruby"],
```

This isn't super obvious, but `projectTags` maps to [these tags](https://github.com/newrelic/opensource-website/blob/develop/src/data/project-tags/project-tags.json), and `ruby` isn't one of them so this had an issue when rendering (thanks, that actually brought this to our attention 😄). Given `ruby` is the primaryLanguage, that can already be used for [filtering on the Explore Projects](https://opensource.newrelic.com/explore-projects/?language=Ruby) page so it feels more appropriate to just drop `ruby` rather than adding it as a new project tag.

```suggestion
  "projectTags": [ "agent" ],
```

Also, this file should be renamed to `newrelic-newrelic-ruby-agent.json` per the [guidelines](https://github.com/newrelic/opensource-website/wiki/Contributing-Project-Data#1-project-metadata-json).